sycl/test/extensions/bindless_images.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %clangxx -S -emit-llvm -fsycl -fsycl-device-only -fsycl-targets=spir64-unknown-unknown %s -o - | FileCheck %s
+
+#include <iostream>
+#include <sycl/sycl.hpp>
+
+// CHECK: spir_kernel void @_ZTSN4sycl3_V16detail19__pf_kernel_wrapperI10image_readEE
+// CHECK: tail call spir_func noundef <4 x float> @_Z17__spirv_ImageReadIDv4_fmiET_T0_T1_
```

Comment:
That is a regular image read. Is it expected?

Reply:
Is it? I believe this is correct, defined [here](https://github.com/intel/llvm/blob/c1ef658791b0104780f715e783db923cbbd80004/libclc/ptx-nvidiacl/libspirv/images/image.cl#L1733)

Reply:
Thanks @isaacault . [here](https://github.com/intel/llvm/blob/c1ef658791b0104780f715e783db923cbbd80004/libclc/ptx-nvidiacl/libspirv/images/image.cl#L1733) implemented the spirv_ImageRead to call https://github.com/intel/llvm/blob/c1ef658791b0104780f715e783db923cbbd80004/libclc/ptx-nvidiacl/libspirv/images/image.cl#L1646C12-L1646C24 _nvvm_sul . We may need to do similar implementation somewhere, but that can be in a follow up PR? @MrSidims .

Reply:
As for your last commit, the Windows behaviour is expected - mangling happening [here](https://github.com/intel/llvm/blob/c1ef658791b0104780f715e783db923cbbd80004/libclc/ptx-nvidiacl/libspirv/images/image.cl#L20-L26).